packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -302,7 +303,7 @@ async function commentsRejectSendPMAsync (comment: DbComment, currentUser: DbUse
     const tag = await Tags.findOne(comment.tagId)
     if (tag) {
       contentTitle = tag.name
-      rejectedContentLink = `<a href="https://lesswrong.com/tag/${tag.slug}/discussion?commentId=${comment._id}">comment on ${tag.name}</a>`
+      rejectedContentLink = `<a href=${tagGetDiscussionUrl({slug: tag.slug}, true)}` + `commentId=${comment._id}">comment on ${tag.name}</a>`
```

Comment:
This was absolute before; after this change it's no longer absolute. I'm not entirely sure why it was absolute, but one hypothesis is that people do moderation from localhost servers and if this wasn't absolute the rejected-content-notification recipient would get a localhost link instead of a lesswrong link.

Reply:
This is still absolute, that's what the second argument "true" does.